hack/e2e.sh

```diff
@@ -104,7 +118,7 @@ export VSPHERE_USERNAME="${GOVC_USERNAME:-}"
 export VSPHERE_PASSWORD="${GOVC_PASSWORD:-}"
 export E2E_CONF_FILE="${REPO_ROOT}/test/e2e/config/vsphere.yaml"
 export E2E_CONF_OVERRIDE_FILE=""
-export E2E_VM_OPERATOR_VERSION="${VM_OPERATOR_VERSION:-v1.8.6-0-gde75746a}"
+export E2E_VM_OPERATOR_VERSION="${VM_OPERATOR_VERSION:-v1.8.6-0-gde75746a-65e87004}"
```

Comment:
Intentional?

Reply:
Also. Shouldn't this break ci? Should we cleanup an image somewhere?

Reply:
This is compatible to vsphere 8 and 7, but good catch, should not be in this PR.

Reply:
Yes.

Reply:
Thank you!